### PR TITLE
Tweak: Revision of miners' and explorers' gear.

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Revision of gear on miner and explorer vendor, and some other justify changes. Lets start:

Explorers:
- Toolbelt for 500 points now copy the insides of miner webbing. In other word, he is empty now. Doesnt affect starter full-loaded toolbelt.

+ Explorer vendor now have Survival Medipen, Kinetic Accelerator and Whiskey. Also added exosuit jetpack and exosuit grenade launcher for those who want tear reality on ripley. Exosuit jetpack slight cheaper, because you need at least 2 for human speed.
+ Explorer closet now contain Explorer suit(miner one) for pressured battles.
+ Kinetic pistol now cheaper, 500 instead of 750. Vendor section "Kinetic Accelerator" change name into "Kinetic weapon". 

Miners:

- Whiskey was removed from vendor. Now its explorers property.

+ Vendor have new section "Modsuits and Exosuits", where you can now obtain all items, possible for buy as explorer. Doesnt contain exosuit jetpack, because he is useless on Lavaland.
+ Added Real Man ID Sticker and myphril coin for match explorer vendor goods.
+ Change Brute medkit into first aid and advanced one to match explorer`s medicine.

Overall:
- Shuffled the vendor goods into different sections that suit them better in my opinion. For example, transfer point card now nearby space cash, all MODsuit related goes into MODsuit section, such as MGL or jetpack modules, etc.
- Upgraded resonator are cheaper, from 2500 to 1500. Extremely underused item, might look at it later.
- Night vision, Clamp and GPS modules was removed. If you want a normal modules - you have Robotic.

## Why It's Good For The Game

When the explorers were introduced, I remained silent, for I was busy. 
But now it's time to scrutinize their arsenal and gameplay. After ~30 hours of careful study and comparison with mining gameplay, i can say that the explorers have almost the same gameplay: kill mobs, collect loot, don't forget about garbage to gain points. Since the professions are essentially similar, I did a fair (in my opinion) review of the arsenal. Removed superfluous that they should not be in my opinion, added what I think is missing, updated the arsenal of miners.
Why? I dont fell justice between gear difference. I dont want to envy those space rats. And i dont want to envy those earth rats for their cool medicine.

## Images of changes

Gear and Exosuits comparison between vendors(outdated, Night vision, Clamp and GPS modules was removed due to PR progress).
image
Consumables section 
image
Kinetic weapon section
image
Mescellanious section
image
Obviously(for me) missing explorer suit
image

## Testing
Reshuffle code, dont forget about missing ,,,,,,, , looks for myself in local server.

<hr>

### Declaration
- [ x] I confirm that I either do not require [pre-approval](/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

<hr>

## Changelog
:cl:
tweak: Revision of starter and vendor gear for both miners and explorers. For more detailed information welcome here https://github.com/ParadiseSS13/Paradise/pull/26525.
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
Fairly sure this may classify as 
> Changes with wide-ranging balance or design implications

as per [pre-approval guidelines](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval), meaning you may need 
>I have obtained such approval and have included a screenshot to demonstrate this below.

**Comment by tipoima** [comment]
Explorers get advanced storage as an anti-frustration measure more than anything. 
It doesn't really affect the balance, since chances are - average Explorer would either loot a bloodred in the first 15 minutes, or drag a locker for all of their bulky loot anyway. But it makes the already tedious inventory management even more so - mining bag, salvage bag, medkit, O2 box, and suddenly all that space is gone.
 
 _The advanced storage module is also on Paramedic._

**Comment by Scribble-Sheep** [comment]
> Whiskey was removed from vendor. Now its explorers property.
What was the logic behind this one?

**Comment by ErdGinalD** [comment]
> > Whiskey was removed from vendor. Now its explorers property.
> 
> What was the logic behind this one?

Explorers doesnt have their alcohol, so whickey is good choice to add it to explorers. Just fun change, so miners have abscinte and explorers - whickey

**Comment by matttheficus** [comment]
Objecting on the storage change and the fact that a megafauna tweak is being included in this PR.

please make atomic PRs

**Comment by ErdGinalD** [comment]
> Objecting on the storage change and the fact that a megafauna tweak is being included in this PR.
> 
> please make atomic PRs

Removing all armor-related changes and drake buff for separate PR change.

**Comment by Scribble-Sheep** [comment]
Also uh
> Doesnt contain exosuit jetpack, because it is useless on Lavaland.

Was nice to have for antaggery, but eh.

**Comment by Scribble-Sheep** [comment]
> > > Whiskey was removed from vendor. Now its explorers property.
> > 
> > 
> > What was the logic behind this one?
> 
> Explorers doesnt have their alcohol, so whickey is good choice to add it to explorers. Just fun change, so miners have abscinte and explorers - whickey

Kinda sucks bc my miner character drinks exclusively whiskey, but eh, that's a personal gripe.

**Comment by ErdGinalD** [comment]
> Also uh
> 
> > Doesnt contain exosuit jetpack, because it is useless on Lavaland.
> 
> Was nice to have for antaggery, but eh.

i agree with that, but for strict miner gaming on lavaland he doesnt make any sence, so i add it only to explorers as fairly logical addition

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -31,33 +31,42 @@
...
-		EQUIPMENT("Point Transfer Card", /obj/item/card/mining_point_card, 500),
+		EQUIPMENT("Mining Charge Detonator", /obj/item/detonator, 150),
 		EQUIPMENT("Shelter Capsule", /obj/item/survivalcapsule, 400),
+		EQUIPMENT("Luxury Shelter Capsule", /obj/item/survivalcapsule/luxury, 3000),
 		EQUIPMENT("Stabilizing Serum", /obj/item/hivelordstabilizer, 400),
 		EQUIPMENT("Survival Medipen", /obj/item/reagent_containers/hypospray/autoinjector/survival, 500),
 	)
-	prize_list["Kinetic Accelerator"] = list(
+	prize_list["Kinetic Weapon"] = list(
 		EQUIPMENT("Kinetic Accelerator", /obj/item/gun/energy/kinetic_accelerator, 750),
```
Comment (BiancaWilkson) [inline code/modules/mining/machine_vending.dm:68]:
```suggestion
	prize_list["Kinetic Weapons"] = list(
```

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -383,37 +390,40 @@
...
-		EQUIPMENT("Point Transfer Card", /obj/item/card/mining_point_card, 500),
+		EQUIPMENT("Fulton Beacon", /obj/item/fulton_core, 400),
 		EQUIPMENT("Stabilizing Serum", /obj/item/hivelordstabilizer, 400),
+		EQUIPMENT("Survival Medipen", /obj/item/reagent_containers/hypospray/autoinjector/survival, 500),
 	)
 
-	prize_list["Kinetic Accelerator"] = list(
-		EQUIPMENT("Kinetic Pistol", /obj/item/gun/energy/kinetic_accelerator/pistol, 750),
+	prize_list["Kinetic Weapon"] = list(
+		EQUIPMENT("Kinetic Accelerator", /obj/item/gun/energy/kinetic_accelerator, 750),
```
Comment (BiancaWilkson) [inline code/modules/mining/machine_vending.dm:425]:
```suggestion
	prize_list["Kinetic Weapons"] = list(
```

**Review by BiancaWilkson — COMMENTED** [review]
Just some very small grammar fixes, code looks good otherwise

File: code/modules/mining/machine_vending.dm [hunk 3]
```diff
@@ -383,37 +390,40 @@
...
 		EQUIPMENT("Fulton Pack", /obj/item/extraction_pack, 1000),
-		EQUIPMENT("Point Transfer Card", /obj/item/card/mining_point_card, 500),
+		EQUIPMENT("Fulton Beacon", /obj/item/fulton_core, 400),
 		EQUIPMENT("Stabilizing Serum", /obj/item/hivelordstabilizer, 400),
+		EQUIPMENT("Survival Medipen", /obj/item/reagent_containers/hypospray/autoinjector/survival, 500),
 	)
 
-	prize_list["Kinetic Accelerator"] = list(
-		EQUIPMENT("Kinetic Pistol", /obj/item/gun/energy/kinetic_accelerator/pistol, 750),
+	prize_list["Kinetic Weapon"] = list(
```
Comment (BiancaWilkson) [inline code/modules/mining/machine_vending.dm:424]: Kinetic weapon should be plural here

File: code/modules/mining/machine_vending.dm [hunk 4]
```diff
@@ -31,33 +31,42 @@
...
 		EQUIPMENT("Industrial Mining Charge", /obj/item/grenade/plastic/miningcharge, 500),
-		EQUIPMENT("Point Transfer Card", /obj/item/card/mining_point_card, 500),
+		EQUIPMENT("Mining Charge Detonator", /obj/item/detonator, 150),
 		EQUIPMENT("Shelter Capsule", /obj/item/survivalcapsule, 400),
+		EQUIPMENT("Luxury Shelter Capsule", /obj/item/survivalcapsule/luxury, 3000),
 		EQUIPMENT("Stabilizing Serum", /obj/item/hivelordstabilizer, 400),
 		EQUIPMENT("Survival Medipen", /obj/item/reagent_containers/hypospray/autoinjector/survival, 500),
 	)
-	prize_list["Kinetic Accelerator"] = list(
+	prize_list["Kinetic Weapon"] = list(
```
Comment (BiancaWilkson) [inline code/modules/mining/machine_vending.dm:67]: And here

File: code/modules/mining/machine_vending.dm (hunk 3 above)
Comment (ErdGinalD) [reply code/modules/mining/machine_vending.dm]: donezo

File: code/modules/mining/machine_vending.dm (hunk 4 above)
Comment (ErdGinalD) [reply code/modules/mining/machine_vending.dm]: donezo

**Review by BiancaWilkson — APPROVED** [review]

**Comment by Qwertytoforty** [comment]
Holding objection for 2 things from me:
Remove:
		EQUIPMENT("Night Vision Module", /obj/item/mod/module/visor/night, 1500),
		EQUIPMENT("Clamp Module", /obj/item/mod/module/clamp, 500),
		EQUIPMENT("GPS Module", /obj/item/mod/module/gps, 500),
As you should have to go to robotics to get modules (jetpacks accepted.) Doubly so for nightvision, as it wasn't included in modsuit production (mainly to limit night vision without losing flash protection and making more modules needed or surgery for a sec hud implant)
Undo:
Removal of advanced jetpack from mining vendor.
As explorers can not get as many points right now, it should be kept in the mining vendor to keep rnd levels more of the same.
Not against this in the future if rnd is reworked to not use this system

**Comment by ErdGinalD** [comment]
> Holding objection for 2 things from me: Remove: EQUIPMENT("Night Vision Module", /obj/item/mod/module/visor/night, 1500), EQUIPMENT("Clamp Module", /obj/item/mod/module/clamp, 500), EQUIPMENT("GPS Module", /obj/item/mod/module/gps, 500), As you should have to go to robotics to get modules (jetpacks accepted.) Doubly so for nightvision, as it wasn't included in modsuit production (mainly to limit night vision without losing flash protection and making more modules needed or surgery for a sec hud implant) Undo: Removal of advanced jetpack from mining vendor. As explorers can not get as many points right now, it should be kept in the mining vendor to keep rnd levels more of the same. Not against this in the future if rnd is reworked to not use this system

Jetpack was moved to "Modsuits and Exosuits" section, as those related item.
GPS, night and clamp will be removed, yeah.

**Comment by Danchi299** [comment]
well, that link feels pointless considering all merged PRs come with this button
image

**Pull request merged**